includes/class-sensei-grading.php

```diff
@@ -1196,7 +1221,8 @@ public static function get_course_users_grades_sum( $course_id ) {
 		$comment_query_piece['from']   = " FROM {$wpdb->comments}  INNER JOIN {$wpdb->commentmeta}  ON ( {$wpdb->comments}.comment_ID = {$wpdb->commentmeta}.comment_id ) ";
 		$comment_query_piece['where']  = " WHERE {$wpdb->comments}.comment_type IN ('sensei_course_status') AND ( {$wpdb->commentmeta}.meta_key = 'percent') AND {$wpdb->comments}.comment_post_ID = {$clean_course_id} ";
 
-		$comment_query     = $comment_query_piece['select'] . $comment_query_piece['from'] . $comment_query_piece['where'];
+		$comment_query = $comment_query_piece['select'] . $comment_query_piece['from'] . $comment_query_piece['where'];
+		// WPCS: unprepared SQL OK.
```

Comment:
Probably you can guess what I would say about this. 😆

Reply:
Soo, I didn't touch this code, I just saw a linter error for this code. I had some general issues with linter so I wanted to minimize the warnings, but I can remove the comment.